backend/src/contaxy/managers/auth.py

```diff
@@ -811,7 +818,10 @@ def login_callback(
 
     # User Operations
 
-    def list_users(self) -> List[User]:
+    def list_users(
+        self,
+        access_level: Optional[AccessLevel] = None,
+    ) -> List[User]:
```

Comment:
Needs to be updated to also return UserRead object (e.g. `Union[User,UserRead]`). This would also need to be updated in the operations and client class.